src/lib/layouts/main/Footer.svelte

```diff
@@ -0,0 +1,218 @@
+<script>
```

Comment:
wondering if can componentize the footer further. I could see us having to edit this with some frequency, especially if we continue to list out current and past studies. Let's leave it as is, for now. But when Meghana and Sydette get to thinking about site content, we may want to adjust the approach here.

Reply:
I hear you and agree. Will refactor.